corehq/apps/app_manager/models.py

```diff
@@ -4381,8 +4383,9 @@ def jad_settings(self):
             settings['Build-Number'] = self.version
         return settings
 
-    def create_build_files(self, build_profile_id=None):
-        all_files = self.create_all_files(build_profile_id)
+    def create_build_files(self, build_profile_id=None, version_reverted_to=None):
```

Comment:
Using past tense in this parameter name (`version_reverted_to`) is interesting, maybe a little surprising. My guess at what it means is something like

> version that was used in an earlier _revert_ operation

Is that a correct interpretation?

I'm guessing it's not, and in fact this is the version that will be used in a future operation (which is about to be performed as a result of this method call). If that is a correct then what do you think of renaming it to `revert_to_version` or even more simply `version`?

In any case a docstring explaining the meaning of that parameter would be useful.